package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zeppelin-solidity",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Why change the version? The version is changed only on each release I think and by the maintainers of the repo, please remove this change.

Reply:
Done